server/views/components/customDisplay/crm14/crm14EvidencePart2.njk

```diff
@@ -14,49 +14,70 @@
 
     {# Render "Processed Attachments" subsection #}
 
-    <div class="govuk-!-margin-top-8 govuk-!-margin-bottom-0">
-        <h4>Previously Submitted and Processed Evidence Files</h4>
-    </div>
+    {% if (data.processedAttachments.length > 0) %}
+        <div class="govuk-!-margin-top-8 govuk-!-margin-bottom-0">
+            <h4>Previously Submitted and Processed Evidence Files</h4>
+        </div>
 
-    {% for item in data.processedAttachments %}
+        {% for item in data.processedAttachments %}
 
-        {% set sectionTitle = 'Evidence File ' ~ loop.index %}
+            {% set sectionTitle = 'Evidence File ' ~ loop.index %}
 
-        {% set fields = [
-            { label: "File name", value: item.fileName },
-            { label: "Evidence type", value: item.evidenceType },
-            { label: "Date submitted", value: item.dtSubmitted, type: "date" },
-            { label: "Status", value: item.status },
-            { label: "Provider notes", value: item.providerNotes },
-            { label: "Caseworker notes", value: item.caseworkerNotes },
-            { label: "Date processed", value: item.dtProcessed, type: "date" }
-        ] %}
+            {% set fields = [
+                { label: "File name", value: item.fileName },
+                { label: "Evidence type", value: item.evidenceType },
+                { label: "Date submitted", value: item.dtSubmitted, type: "date" },
```

Comment:
Can we change it to `dateSubmitted`

Reply:
Unfortunately, this is the field name in the BE API.
If you want this changed, please can you raise a separate ticket so that it can be changed in the backend first.